src/mongo/services/MongoCodeLensProvider.ts

```diff
@@ -61,8 +61,7 @@ export class MongoCodeLensProvider implements vscode.CodeLensProvider {
                     lenses.push(<vscode.CodeLens>{
                         command: {
                             title: "Execute",
-                            command: 'cosmosDB.executeMongoCommand',
-                            arguments: [cmd.text]
```

Comment:
`executeCommandFromActiveEditor` decides which command to run based on the cursor location, but that doesn't always correspond to the command being passed here in the codelens. You need to pass along _something_ here so that you know which command to execute. Can you pass along the position of the codelens? If so, that seems easy enough to do - it would replace the position used [here](https://github.com/microsoft/vscode-cosmosdb/blob/master/src/mongo/MongoScrapbook.ts#L52)